src/redux/Reducers/setting.js

```diff
@@ -1,9 +1,11 @@
 import { defineState } from "redux-localstore";
 
-import { SET_LANGUAGE } from "../Actions/index.js";
+import { defaultTheme } from "../../utils/constants.js";
```

Comment:
can we add a config variable e.g. `theme_select` of type `"auto" | <themename>` where auto detects the users [preferred theme](https://stackoverflow.com/questions/56393880/how-do-i-detect-dark-mode-using-javascript) and the theme name sets the one as default.

Reply:
The env variable `REACT_APP_VOCASCAN_THEME_SELECT` is now supported and added to documentation in the docs PR